lib/xdparser2.ts

```diff
@@ -676,92 +676,104 @@ function parseSplitsFromAnswer(answerWithSplits: string, splitCharacter?: string
 
 export function inlineMarkdownParser(str: string): MDClueComponent[] {
   const components: MDClueComponent[] = []
-  let token = ""
-  let mode: MDClueComponent[0] = "text"
-  let linkText = ""
-
-  const pushText = () => {
-    if (token.length > 0) {
-      components.push(["text", token])
-      token = ""
+  const mdTokens = ["**", "\\\\", "~~", "[", "]", "(", ")"]
+  const getMDType = (operator: string) => {
+    switch (operator) {
+      case "**": return "bold"
+      case "\\\\": return "italics"
```

Comment:
Long story, but it should be `//` not `\\` kinda interested to know where that came from too - looks like a discussion in here which I'm also gonna be blocking these changes on: https://github.com/puzzmo-com/games/issues/1709

it should be markdown-ish but the doubles trigger formatting for backwards compatibility

Reply:
Seems to be that `//` appears naturally in `https://www.` - so I think we make the parser always require the space before to start for all tokens

Reply:
yea i'll mention this in the issue. i'll change things to do `//`

Reply:
BBCode would be better than HTML, either would be better than what we have now. BBCode is superior because we don't have to worry about escaping it crossword editor where we may want to display the contents of an xd file. Another benefit of BBCode is that it is the least likely to collide with a crossword constructor using the the syntax unintentionally in a clue somewhere (HTML probably also safe in this case though). So, in BBCode style, we'd have:

- [b]{text}[/b]
- [i]{text}[/i]
- [s]{text}[/s]
- [url={url}]{text}[/url]

From https://www.bbcode.org/reference.php

Reply:
ok this is cool. first time i've seen bbcode 👍